diff --git a/tools/package_binaries.py b/tools/package_binaries.py
index 9bd2e152..b5faff36 100755
--- a/tools/package_binaries.py
+++ b/tools/package_binaries.py
@@ -220,7 +220,10 @@ def generate_target_nw(platform_name, arch, version):
             target['input'].append('chromedriver')
             target['input'].append('libffmpeg.dylib')
             target['input'].append('minidump_stackwalk')
-            target['input'].append('v8_context_snapshot.x86_64.bin')
+            if arch == 'x64':
+                target['input'].append('v8_context_snapshot.x86_64.bin')
+            elif arch == 'arm64':
+                target['input'].append('v8_context_snapshot.arm64.bin')
     else:
         print ('Unsupported platform: ' + platform_name)
         exit(-1)
